content/SCALE/SCALETutorials/Apps/UsingApps.md

```diff
@@ -78,7 +65,7 @@ To deploy a custom application container in the SCALE web interface, go to **App
 
 Custom applications use the system-level Kubernetes Node IP settings by default. You can assign an external interface to custom apps by setting one on the **Networking** section of the **Launch Docker Image** form. 
 
-Unless you need to run an application separately from the Web UI, we recommend using the default Kubernetes Node IP (0.0.0.0) to ensure apps function properly.
+Unless you need to run an application separately from the Web UI, we recommend using the default Kubernetes Node IP (0.0.0.0) to ensure apps function correctly.
```

Comment:
I think you are specifically talking about the UI setting so suggesting this change:
```suggestion
Unless you need to run an application separately from the Web UI, we recommend using the default Kubernetes **Node IP** (0.0.0.0) to ensure apps function correctly.
```

Reply:
If you are not intending to specifically mention the UI setting, then change Node IP to node IP